ansible/configs/ans-tower-prod/pre_software.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Step 003 - Create env key
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tags:
+    - step003
+    - generate_env_keys
+  tasks:
+    - name: Generate SSH keys
+      shell: ssh-keygen -b 2048 -t rsa -f "{{output_dir}}/{{env_authorized_key}}" -q -N ""
```

Comment:
i think there is a role that manages env keys?

Reply:
The role is not creating any directories and ssh keys on local host. I have included the set_env_authorized_key role for all hosts.